### PR TITLE
UI / CSS tweaks

### PR DESCRIPTION
**Pull request by csillag**
Small tweaks based on input from @donouwens 

Before:

image

After:

image

### CODE REVIEW

File: src/app/components/PageLayout/Footer.tsx [hunk 1]
```diff
@@ -16,6 +16,9 @@ const FooterBox = styled(Box)(({ theme }) => ({
     flex: '0 1 100%',
     padding: `${theme.spacing(5)} ${theme.spacing(6)}`,
   },
+  position: 'absolute',
+  bottom: 0,
```
Comment (csillag) [inline src/app/components/PageLayout/Footer.tsx:20]: Nah, this causes problems for long pages.

Comment (csillag) [reply src/app/components/PageLayout/Footer.tsx]: We are all good now, thanks to a tip from @lubej .

**Review by buberdds — DISMISSED** [review]

File: src/app/components/PageLayout/index.tsx [hunk 2]
```diff
@@ -22,10 +22,11 @@ export const PageLayout: FC<PropsWithChildren<PageLayoutProps>> = ({ children, m
           pt: 4,
           px: isMobile ? 0 : '4%',
           border: isMobile ? 'none' : `solid 15px ${theme.palette.layout.border}`,
+          minHeight: '100vh',
         }}
       >
         <Header />
-        <main>{children}</main>
+        <main style={{ minHeight: '75vh' }}>{children}</main>
```
Comment (buberdds) [inline src/app/components/PageLayout/index.tsx:29]: styled

Comment (buberdds) [reply src/app/components/PageLayout/index.tsx:29]: Does it work correctly on mobile?

Comment (buberdds) [reply src/app/components/PageLayout/index.tsx:29]: yep, not able to repro mobile issue

**Review by buberdds — APPROVED** [review]

**Pull request merged**